packages/foundations-ts-definitions/types/index.ts

```diff
@@ -7,3 +7,4 @@ export * from './marketplace-schema'
 export * from './payments-schema'
 // @ts-ignore
 export * from './platform-schema'
+export * from './webhook-schema'
```

Comment:
See below, you need to remove this line and add the webhook schema here https://github.com/reapit/foundations/blob/master/packages/foundations-ts-definitions/scripts/create-index-file.js#L19